app/assets/stylesheets/earthworks.css

```diff
@@ -134,26 +134,48 @@
   outline: none;
 }
 
+.search-q .input-group {
+  padding: .5rem
+}
+
 .input-group > .search-autocomplete-wrapper {
   border: none;
 }
 
-.search-query-form, .navbar-search .search-query-form{
+.input-group > .search-autocomplete-wrapper input {
+  padding-top: 0;
+  padding-bottom: 0
+}
+
+.search-query-form, .navbar-search .search-query-form {
   border: 1px solid #ABABA9;
-  padding: .5rem;
+  padding: 8px;
```

Comment:
These are equivalent.  Can you revert this so the design will be relative to the base font height?